### DISCUSSION TITLE
Альтернативы gui.py

### DISCUSSION THREAD

**Discussion by samorez1** [rendrom/rosreestr2coord#89] [General]
Добрый вечер. Не подскажите, что можно использовать в качестве альтернативы gui.py? С ним было очень удобно) Заранее спасибо!

**Comment by rendrom** [comment]
Ого, ничего себе вы вспомнили. Сейчас в этом репозитории остался только консольный и программный режимы. Нет планов по созданию графического интерфейса. Можете посмотреть по форкам и по другим репозиториям где rosreestr2coord в зависимостях
